Merge queue crashed

Merge queue just crashed with the following error:

E0520 07:57:15.093393       1 github.go:1155] Can't load commit kubernetes kubernetes bf1f3d37206b0a69c42b8df64c84942c07d5fc3e: GET https://api.github.com/repos/kubernetes/kubernetes/commits/bf1f3d37206b0a69c42b8df64c84942c07d5fc3e: 500  []
E0520 07:57:15.133638       1 github.go:899] Error getting events for issue: GET https://api.github.com/repos/kubernetes/kubernetes/issues/25921/events?page=1&per_page=100: 500  []
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x4a4572]

goroutine 1 [running]:
panic(0x9910e0, 0xc820014060)
	/usr/local/google/home/dbsmith/.gvm/gos/go1.6/src/runtime/panic.go:464 +0x3e6
k8s.io/contrib/mungegithub/github.(*MungeObject).LabelCreator(0xc82cacea10, 0xc82000be26, 0xc, 0x0, 0x0)
	/usr/local/google/home/dbsmith/code/k3/src/k8s.io/contrib/mungegithub/github/github.go:577 +0x62
k8s.io/contrib/mungegithub/mungers.(*PathLabelMunger).Munge(0xc820112570, 0xc82cacea10)
	/usr/local/google/home/dbsmith/code/k3/src/k8s.io/contrib/mungegithub/mungers/path_label.go:153 +0x5d6
k8s.io/contrib/mungegithub/mungers.MungeIssue(0xc82cacea10, 0x0, 0x0)
	/usr/local/google/home/dbsmith/code/k3/src/k8s.io/contrib/mungegithub/mungers/mungers.go:121 +0x9a
k8s.io/contrib/mungegithub/github.(*Config).ForEachIssueDo(0xc82006db00, 0xb597a0, 0x0, 0x0)
	/usr/local/google/home/dbsmith/code/k3/src/k8s.io/contrib/mungegithub/github/github.go:1551 +0xc56
main.doMungers(0xc82006db00, 0x0, 0x0)
	/usr/local/google/home/dbsmith/code/k3/src/k8s.io/contrib/mungegithub/mungegithub.go:65 +0x20a
main.main.func1(0xc820082e00, 0xc820112750, 0x0, 0x3, 0x0, 0x0)
	/usr/local/google/home/dbsmith/code/k3/src/k8s.io/contrib/mungegithub/mungegithub.go:106 +0x30f
k8s.io/contrib/mungegithub/vendor/github.com/spf13/cobra.(*Command).execute(0xc820082e00, 0xc82000a290, 0x3, 0x3, 0x0, 0x0)
	/usr/local/google/home/dbsmith/code/k3/src/k8s.io/contrib/mungegithub/vendor/github.com/spf13/cobra/command.go:561 +0x62c
k8s.io/contrib/mungegithub/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc820082e00, 0xc820082e00, 0x0, 0x0)
	/usr/local/google/home/dbsmith/code/k3/src/k8s.io/contrib/mungegithub/vendor/github.com/spf13/cobra/command.go:651 +0x55c
k8s.io/contrib/mungegithub/vendor/github.com/spf13/cobra.(*Command).Execute(0xc820082e00, 0x0, 0x0)
	/usr/local/google/home/dbsmith/code/k3/src/k8s.io/contrib/mungegithub/vendor/github.com/spf13/cobra/command.go:610 +0x2d
main.main()
	/usr/local/google/home/dbsmith/code/k3/src/k8s.io/contrib/mungegithub/mungegithub.go:124 +0x395

@lavalamp @eparis @gmarek

--- a/mungegithub/github/github.go
+++ b/mungegithub/github/github.go
@@ -599,7 +599,7 @@ func (obj *MungeObject) LabelTime(label string) *time.Time {
 // LabelCreator returns the login name of the user who (last) created the given label
 func (obj *MungeObject) LabelCreator(label string) string {
 	event := obj.labelEvent(label)
-	if event == nil {
+	if event == nil || event.Actor == nil || event.Actor.Login == nil {
 		return ""
 	}
 	return *event.Actor.Login

